lib/manageiq/api/client/connection.rb

```diff
@@ -103,7 +103,10 @@ def send_request(method, path, params, &block)
         end
 
         def check_response
-          if response.status >= 400
+          if response.status == 404
+            message = json_response.fetch_path("error", "message") || json_response["error"]
```

Comment:
I think this is fine, I just don't remember, when is json response populated as { "error" : "....." } vs. { "error" : { "kind" : "...", "message" : ".... " }, ...}

Reply:
I don't know, I was basing this on the existing examples.